USERS.md

```diff
@@ -189,6 +189,13 @@ Users (Alphabetically)
       U: L3/L4 policies, Hubble, BPF NodePort, CiliumClusterwideNetworkPolicy
       Q: @Eric Bailey, @Ole Markus
 
+    * N: Talos
+      D: Cilium is one of the supported CNI's in Talos
+      U: Networking, NetworkPolicy
+      L: https://slack.dev.talos-systems.io/
```

Comment:
Would you have a link to something that doesn't require subscribing before it can be accessed? Ideally, something that also mentions the use of Cilium.

Reply:
fixed